### ISSUE TITLE
Long import times

### ISSUE THREAD

**Issue by mrocklin** [dask/distributed#716]
Import times have risen significantly recently

```python
In [1]: %time import distributed
CPU times: user 880 ms, sys: 240 ms, total: 1.12 s
Wall time: 961 ms
```

Only a modest fraction of this is from our historically large dependencies

```python
In [1]: %time import numpy, pandas, tornado, bokeh
CPU times: user 404 ms, sys: 240 ms, total: 644 ms
Wall time: 433 ms

In [2]: %time import distributed
CPU times: user 588 ms, sys: 44 ms, total: 632 ms
Wall time: 647 ms
```

**Comment by jcrist** [comment]
Some more timings:

```bash
(dask) jcrist dask $ time python -c "print('baseline')"
baseline

real    0m0.022s
user    0m0.013s
sys     0m0.008s
(dask) jcrist dask $ time python -c "import distributed"

real    0m0.851s
user    0m0.676s
sys     0m0.172s
(dask) jcrist dask $ time python -c "import dask"

real    0m0.644s
user    0m0.513s
sys     0m0.127s
(dask) jcrist dask $ time python -c "import numpy, pandas"

real    0m0.581s
user    0m0.469s
sys     0m0.103s
```

**Comment by pitrou** [comment]
Here:
```
$ time python -c "import distributed"

real	0m0.963s
user	0m0.892s
sys	0m0.060s
$ time python -c "import dask, numpy, pandas, tornado"

real	0m0.486s
user	0m0.432s
sys	0m0.040s
```

That's indeed a large overhead for distributed.

**Comment by pitrou** [comment]
Module list: https://gist.github.com/pitrou/098375cb17193c95fb378cae82fc5859

**Comment by pitrou** [comment]
Also see https://github.com/pytoolz/cytoolz/pull/93

**Comment by mrocklin** [comment]
Fixed by #718
